cds_ils/importer/providers/cds/rules/book.py

```diff
@@ -39,3 +41,31 @@ def alternative_titles(self, key, value):
                 }
             )
         return _alternative_titles
+
+
+@model.over("_migration", "^340__")
+@out_strip
+def medium(self, key, value):
+    """Translates medium."""
+    _migration = self.get("_migration", {})
+    _item_medium = _migration.get("item_medium", [])
+    barcodes = force_list(value.get("x", ""))
+    added_barcodes = []
+    _medium = clean_val("a", value, str)
+    if barcodes[0]:
+        for barcode in barcodes:
+            if barcode not in added_barcodes:
+                _item_medium.append(
+                    {
+                        "barcode": barcode,
+                        "medium": _medium,
```

Comment:
Medium field in the items is a vocabulary, should we map the values ?
For instance, in this [record](https://cds.cern.ch/record/280876/export/xm?ln=en) the medium is CD-ROM, while we should map it to CDROM, as it is in the [vocabularies](https://github.com/CERNDocumentServer/cds-ils/blob/master/cds_ils/vocabularies/data/item_mediums.json#L13).